native/src/utils/calendarRangeUtils.ts

```diff
@@ -0,0 +1,61 @@
+import { DateTime } from 'luxon'
+
+import { lightColors } from 'build-configs/integreat/theme/colors'
+
+type MarkedDateType = {
+  selected: boolean
+  startingDay?: boolean
+  endingDay?: boolean
+}
+
+export const getMarkedDates = (
+  startDate: DateTime | null,
+  endDate: DateTime | null,
+): Record<string, MarkedDateType> => {
+  const markedDateStyling = {
+    color: lightColors.themeColor,
+    textColor: lightColors.textColor,
+  }
+
+  const markedDates: Record<string, MarkedDateType> = {}
+
+  const cutoffDate = DateTime.fromISO('2020-01-01')
```

Comment:
The cutoffDate is just added for cases like fromdate: 2006 for example this will crash the app because of the marking can't handle all that in the memory 😂 so anything before 2020 will not be marked. 

In the case of DateTime.now() if I choose a day before today it will not be selected and this could create a confusion for the user.
How about a minus year from now... This makes since I guess.

Reply:
Alright, then feel free to keep this cutoff date and maybe just add a comment to explain :)